pages/register-trees.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import dynamic from 'next/dynamic';
+import Head from 'next/head';
+
+interface Props {}
+
+export default function Register({}: Props): ReactElement {
+  const RegisterTrees = dynamic(
+    () => import('../src/tenants/treesforjane/RegisterTrees/JaneRegisterTrees')
```

Comment:
If we ever should decide it would be cool to allow unregistered users to register trees and want to takeover this into the main application, we must move the files out of the tenant folder.